### PR TITLE
Implemented LastPhotos.jsx component ( ProfilePage.jsx )

### LINKED ISSUE #76: Implement Last Photos React UI Component ( ProfilePage.jsx )
**Is your feature request related to a problem? Please describe.**

We currently have an `Last Photos` Component design taken from the [html olympus template](https://html.crumina.net/html-olympus/02-ProfilePage.html) and modified a bit from me. I haven't thought of responsive measurements for now the content inside will be responsive but the actual location of the component when viewed at smaller resolutions is not decided yet. I will update the issue when we do.

**Design:**

design1

**Component location(at the right of the Post Component, below the ProfileHeader Component):**

design2

**Implementation**
In this issue, you are supposed to write html, css code for the Last Photos component. Make a new folder and component in `components/`. For css, you can use `styled-components` and then import them into `LastPhotos.jsx`. Use `ant-design` wherever possible.

You can work on this feature using a branch called `feature/lastphotos-component`.

## Pull Request Squashing
Please, go through these steps before you submit a PR.

1. Make sure that your PR is not a duplicate.
2. If not, then make sure that:

  2.1. You have done your changes in a separate branch. Branches MUST have descriptive names that start with either the `fix/` or `feature/` prefixes. Good examples are: `fix/signin-issue` or `feature/issue-templates`.

  2.2. You have a descriptive commit message with a short title (first line).

  2.3. You have only one commit (if not, squash them into one commit).

  2.4. `yarn test` doesn't throw any error. If it does, fix them first and amend your commit (`git commit --amend`).

3. **After** these steps, you're ready to open a pull request.

  3.1. Give a descriptive title to your PR.

  3.2. Provide a description of your changes.

  3.3. Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### PR DESCRIPTION
**Pull request by KristianWEB**
* Implemented LastPhotos component 
* Closes #76

### CODE REVIEW

File: src/App/components/ProfilePage/components/LastPhotos.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+const LastPhotos = () => {
+  return (
+    <LastPhotosComponent>
+      <HeadingContainer>
+        <LastPhotosHeading>Last Photos</LastPhotosHeading>
+      </HeadingContainer>
+      <LastPhotosContainer>
+        <LastPhoto>
+          <LastPhotoImg src={FirstImage} />
```
Comment (vinodh99) [inline src/App/components/ProfilePage/components/LastPhotos.jsx]: U can make `LastPhoto` as a reusable component and have an object mapped with all the images in a constants folder. Loop through the object and map it using the reusable component.

File: src/App/components/ProfilePage/components/Post.styles.js [hunk 2]
```diff
@@ -18,7 +18,9 @@ export const PostHeader = styled.div`
 export const PostCard = styled(Card)`
   border: none;
   margin: 1rem;
-  border-radius: 8px;
+
+  /** ant-design adds its own border-radius: 3px and somehow I can't change that value so I had to put !important to make it work. Please let me know if you have any ideas to prevent this kind of collision with antd. */
+  border-radius: 8px !important;
```
Comment (vinodh99) [inline src/App/components/ProfilePage/components/Post.styles.js]: Have u tried grabbing the `className` of antd and then override the CSS?

File: src/App/components/ProfilePage/components/LastPhotos.jsx (hunk 1 above)
Comment (KristianWEB) [reply src/App/components/ProfilePage/components/LastPhotos.jsx]: I know. The whole component is a placeholder, I will change that in the future when we have our backend data setup.

Comment (KristianWEB) [reply src/App/components/ProfilePage/components/LastPhotos.jsx]: Tbh all the stuff is placeholders for now. When I have the actual data I'll refactor.

File: src/App/components/ProfilePage/components/Post.styles.js (hunk 2 above)
Comment (KristianWEB) [reply src/App/components/ProfilePage/components/Post.styles.js]: Haven't tried that. Will try it and will let you know!

**Pull request merged**